torch/autograd/graph.py

```diff
@@ -227,8 +227,17 @@ def increment_version(tensor: torch.Tensor) -> None:
 
     Note that incrementing the version counter multiple times for a single inplace operation
     is not problematic.
+
+    Note that if you pass in tensor constructed under torch.inference_mode(),
+    we will not bump its version counter (because your tensor does not have one).
+
+    Can accept either a tensor, or a list of tensors.
+    If you pass in a list of tensors, we will separately
```

Comment:
You stopped mid sentence? :p 
No need for a paragraph for this, you can put it inline above.

Reply:
... oops